bincrafters_conventions/actions/update_a_python_version.py

```diff
@@ -17,8 +17,8 @@ def update_a_python_version(main, path, current_python_version, check_for_old_ve
                     content = ifd.readlines()
                 with open(path, 'w') as ofd:
                     for line in content:
-                        if 'PYTHON:' in line:
-                            line = '    PYTHON: "C:\\\\Python{}"\n'.format(current_python_version)
+                        if 'PYTHON:' in line or 'PYTHON_HOME:' in line:
```

Comment:
this is getting executed before the `update python variable name` script, which seems wrong. The order is important. The script should update everything in one run. if you run this in an directory which has an old python version + the old python variable name, you won't get both updates correctly

Reply:
so, do you think it should be executed after the `update python variable name` script?

Reply:
I guess the best case would be updating the environment variable name before the other AppVeyor Python updates. By updating the variable name first we only need to check for the new name in all scripts afterwards instead of both the old and new name

Reply:
thanks, that sounds logical